### PR TITLE
Fix important issue upon startup

### PR DESCRIPTION
**Pull request by spotlightishere**
ok so imagine this
*story time*

you're a new user
```
git clone https://github.com/Seriell/Yuuki-Bot
cd Yuuki-Bot
bash run_linux.sh
```
but n0
the `data` dir doesn't exist by default and the world is ruined! google doesn't say anything because it's too common and you're panicking because you bet a friend you could get the bot up under 30 seconds but you missed it and well now you're depressed so you go to a psychiatrist who doesn't know what they're doing and makes you feel worse about the github code issue so you become a pro blackhat hacker and crack @Seriell's public key and ruin the entire project whilst your food is getting cold from McDonald's because you're too lazy to go out and buy actual food from the store which I mean has its perks (less burnt food and dependable syntax of cooking) but now you're turning out like an average american or sterotypical or something like that and

tl;dr if you don't merge this nasa will come to your house and crack your ssh key and force merge this or something.

### CODE REVIEW

**Comment by spotlightishere** [comment]
and because I forgot to change branches, you now get the extra bonus of fixing the resulting issue in #15 🎉

**Comment by ArtutoGamer** [comment]
what a great history

**Comment by gnmmarechal** [comment]
Art.
